aspnetcore/blazor/hosting-models.md

```diff
@@ -17,13 +17,15 @@ Blazor is a web framework designed to run client-side in the browser on a [WebAs
 
 ## Blazor WebAssembly
 
-The principal hosting model for Blazor is running client-side in the browser on WebAssembly. The Blazor app, its dependencies, and the .NET runtime are downloaded to the browser. The app is executed directly on the browser UI thread. UI updates and event handling occur within the same process. The app's assets are deployed as static files to a web server or service capable of serving static content to clients. Because the app is created for deployment without a backend ASP.NET Core app, it's called a *standalone Blazor WebAssembly app*.
+The primary Blazor hosting model is running client-side in the browser on WebAssembly. The Blazor app, its dependencies, and the .NET runtime are downloaded to the browser. The app is executed directly on the browser UI thread. UI updates and event handling occur within the same process. The app's assets are deployed as static files to a web server or service capable of serving static content to clients.
 
 ![Blazor WebAssembly: The Blazor app runs on a UI thread inside the browser.](hosting-models/_static/blazor-webassembly.png)
 
-To create a Blazor app using the client-side hosting model, use the **Blazor WebAssembly App** template ([`dotnet new blazorwasm`](/dotnet/core/tools/dotnet-new)).
+When the Blazor WebAssembly app is created for deployment without a backend ASP.NET Core app to serve its files, the app is called a *standalone* Blazor WebAssembly app. When the app is created for deployment with a backend app to serve its files, the app is called a *hosted* Blazor WebAssembly app. A hosted Blazor WebAssembly app typically interacts with the server over the network using web API calls or [SignalR](xref:signalr/introduction) (<xref:tutorials/signalr-blazor-webassembly>).
 
-After selecting the **Blazor WebAssembly App** template, you have the option of configuring the app to use an ASP.NET Core backend by selecting the **ASP.NET Core hosted** check box ([`dotnet new blazorwasm --hosted`](/dotnet/core/tools/dotnet-new)). The ASP.NET Core app serves the Blazor app to clients. An app with an ASP.NET Core backend is called a *hosted Blazor WebAssembly app*. The Blazor WebAssembly app can interact with the server over the network using web API calls or [SignalR](xref:signalr/introduction) (<xref:tutorials/signalr-blazor-webassembly>).
+To create a Blazor WebAssembly app in [Visual Studio](https://visualstudio.microsoft.com), use the **Blazor App** > **Blazor WebAssembly App** project template. After selecting the template, you have the option of configuring the app to use an ASP.NET Core backend by selecting the **ASP.NET Core hosted** check box.
```

Comment:
Not sure whether this is better or not. I'll leave it up to you, @guardrex  to judge.

```suggestion
To create a Blazor WebAssembly app in [Visual Studio](https://visualstudio.microsoft.com), use the **Blazor App** > **Blazor WebAssembly App** project template. After selecting the template, you have the option of enabling the `hosted` model by selecting the **ASP.NET Core hosted** check box. This will also generate a server project in the created solution and add sample API interaction call in the client app.
```

Reply:
I'll edit the suggestion to remove "model" so that it doesn't sound like there's a third hosting model for Blazor. Thus far, we've always only defined "hosting model" as *how components are hosted within an app* (WASM or Server). We haven't split WASM into two further "hosting models" (standalone and hosted).

Now that I reflect on this, I think the document should call that concept out. If we define it explicitly, we should 🤞 be able to avoid some of the overloading of the word "hosted" these days. I'll add that on a future commit and leave a comment with the text that I used.

```suggestion
To create a Blazor WebAssembly app in [Visual Studio](https://visualstudio.microsoft.com), use the **Blazor App** > **Blazor WebAssembly App** project template. After selecting the template, you have the option of selecting the **ASP.NET Core hosted** check box to serve the Blazor WebAssembly app to clients from an ASP.NET Core app. The project template for a hosted Blazor WebAssembly solution includes separate server and client projects. The solution includes a sample web API interaction between the server and client projects.
```

Reply:
Thinking a bit further 🤔 in more detail ...

Thus far based on the original text, "how components are hosted by the app" boils down to how the component C# code is executed ... WASM runtime in the browser versus on the server in an ASP.NET Core app.

Reply:
🤔 ... I like what we have. The rubber :duck: says that we already scope the word "hosting (model)" fairly well in the opening paragraphs. The bits that I add for the difference between standalone and "hosted" WASM clarify that in that context we're talking about deployment scenarios, and I use "deployment" there.

This was probably all considered (and rejected) in ancient Blazor design times, but an "ASP.NET Core hosted" WASM app might have been called an "ASP.NET Core deployed/deployment" WASM app, which may have eliminated the double context of "host/hosted." Anywho ... that's all water under the bridge at this point. :smile:

Reply:
I agree. Please go with your original text.